config/example/node.yaml

```
@@ -1,159 +1,159 @@
-        small_size_limit: 102400
+        path: tmp/0/blob  # blobstor path
+        perm: 0644  # permissions for blobstor files(directories: +x for current user and group)
+        compress: true  # zstd, level 3, unconfigurable right now
+        shallow_depth: 5 # directory structure depth
+        small_size_limit: 102400  # size threshold for objects that are considered to be "small", bytes
 
         blobovnicza:
-          size: 4194304
-          shallow_depth: 1
-          shallow_width: 4
-          opened_cache_size: 50
+          size: 4194304  # coarse one db size limit, total size will be: size*width^(depth+1), bytes
+          shallow_depth: 1  # directory structure depth
+          shallow_width: 4   # directory structure width
+          opened_cache_size: 50  # maximum number of opened database files
 
       gc:
-        remover_batch_size: 150
-        remover_sleep_interval: 2m
+        remover_batch_size: 150  # number of objects to be removed by the garbage collector
+        remover_sleep_interval: 2m  # frequency of the garbage collector invocation
 
     1:
-      refill_metabase: true
+      refill_metabase: true  # sync metabase with blobstore on start, expensive, leave false until complete understanding
 
-      use_write_cache: true
+      use_write_cache: true  # use write-cache
 
       writecache:
-        path: tmp/1/cache
-        mem_size: 2147483648
-        small_size: 16384
-        max_size: 134217728
-        workers_number: 30
-        size_limit: 4294967296
+        path: tmp/1/cache  # write-cache root directory
+        mem_size: 2147483648  # coarse RAM usage limit for "small" objects, bytes
+        small_size: 16384  # size threshold for objects that are considered to be "small", bytes
+        max_size: 134217728  # objects that are bigger than <max_size> bypass write-cache and go to the storage directly, bytes
+        workers_number: 30  # number of write-cache flusher threads
+        size_limit: 4294967296  # coarse write-cache total size, bytes
 
       metabase:
-        path: tmp/1/meta
-        perm: 0644
+        path: tmp/1/meta  # metabase path
+        perm: 0644  # permissions for metabase files(directories: +x for current user and group)
 
       blobstor:
-        path: tmp/1/blob
-        perm: 0644
-        compress: false
-        shallow_depth: 5
-        small_size_limit: 102400
+        path: tmp/1/blob  # blobstor path
+        perm: 0644  # permissions for blobstor files(directories: +x for current user and group)
+        compress: false  # zstd, level 3, unconfigurable right now
+        shallow_depth: 5  # directory structure depth
+        small_size_limit: 102400  # size threshold for objects that are considered to be "small", bytes
 
         blobovnicza:
-          size: 4194304
-          shallow_depth: 1
-          shallow_width: 4
-          opened_cache_size: 50
+          size: 4194304  # coarse one db size limit, total size will be: size*width^(depth+1), bytes
```

I am not sure `coarse` is a suitable word there. Probably direct translation of "грубый", but I haven't seen usage of it in this context.

What about  
```suggestion
          size: 4194304  # approximate size limit of single blobovnicza instance, total size will be: size*width^(depth+1), bytes
```